wp_gutenberg_ref_update_prs.sh

```diff
@@ -0,0 +1,283 @@
+BRANCH_NAME=${BRANCH_NAME:-"$DEFAULT_GB_MOBILE_BRANCH_NAME"}
+if [[ -z "$BRANCH_NAME" ]]; then
+    abort "Version number cannot be empty."
+fi
+
+# Ask for WPAndroid target branch
+read -r -p "Do you want to target $WPANDROID_TARGET_BRANCH branch for WPAndroid PR? (y/n) " -n 1
+echo ""
+if [[ $REPLY =~ ^[Nn]$ ]]; then
+    read -r -p "Enter the branch name you want to target. Make sure a branch with this name already exists in WPAndroid repository: " WPANDROID_TARGET_BRANCH
+fi
+
+# Ask for WPiOS target branch
+read -r -p "Do you want to target $WPIOS_TARGET_BRANCH branch for WPiOS PR? (y/n) " -n 1
+echo ""
+if [[ $REPLY =~ ^[Nn]$ ]]; then
+    read -r -p "Enter the branch name you want to target. Make sure a branch with this name already exists in WPiOS repository: " WPIOS_TARGET_BRANCH
+fi
+
+# Ensure javascript dependencies are up-to-date
+read -r -p "Run 'npm ci' to ensure javascript dependencies are up-to-date? (y/n) " -n 1 PERFORM_NPM_CI
+echo ""
+confirm_to_proceed "Script execution for creating WPApps PR's will take ~15-20 mins. Proceed? "
+if [[ $PERFORM_NPM_CI =~ ^[Yy]$ ]]; then
+    execute "npm" "ci"
+fi
+
+# Create Git branch
+GUTENBERG_MOBILE_BRANCH="$BRANCH_NAME"
+ohai "Create Git branch '$GUTENBERG_MOBILE_BRANCH' in gutenberg-mobile."
+execute "git" "switch" "-c" "$GUTENBERG_MOBILE_BRANCH"
+
+# Create Git branch in Gutenberg
+GB_BRANCH="rnmobile/$GUTENBERG_MOBILE_BRANCH"
+ohai "Create Git branch '$GB_BRANCH' in gutenberg."
+cd gutenberg
+execute "git" "switch" "-c" "$GB_BRANCH"
+cd ..
+
+# Make sure podfile is updated
+ohai "Make sure podfile is updated"
+PRE_IOS_COMMAND="npm run core preios"
+eval "$PRE_IOS_COMMAND"
+
+# If preios results in changes, commit them
+cd gutenberg
+if [[ -n "$(git status --porcelain)" ]]; then
+    ohai "Commit changes from '$PRE_IOS_COMMAND'"
+    execute "git" "commit" "-a" "-m" "WP Integration script: Update with changes from '$PRE_IOS_COMMAND'"
+else
+    ohai "There were no changes from '$PRE_IOS_COMMAND' to be committed."
+fi
+cd ..
+
+# Commit updates to gutenberg submodule
+if [[ -n "$(git status --porcelain)" ]]; then
+    ohai "Commit updates to gutenberg submodule"
+    execute "git" "add" "gutenberg"
+    execute "git" "commit" "-m" "WP Integration script: Update gutenberg ref"
+else
+    ohai "There were no changes from submodule update to be committed."
+fi
+
+# Update the bundles
+ohai "Update the bundles"
+npm run bundle || abort "Error: 'npm bundle' failed.\nIf there is an error stating something like \"Command 'bundle' unrecognized.\" above, perhaps try running 'rm -rf node_modules gutenberg/node_modules && npm ci'."
+
+# Commit bundle changes
+if [[ -n "$(git status --porcelain)" ]]; then
+    ohai "Commit bundle changes"
+    execute "git" "add" "bundle/"
+    execute "git" "commit" "-m" "WP Integration script: Update bundle for: $BRANCH_NAME"
+else
+    ohai "There were no changes from bundle update to be committed."
+fi
+
+
+#####
+# Create PRs
+#####
```

Comment:
@cameronvoell It seemed to me like from this point the script is doing exactly the same thing with `./release_automation.sh`. Do you think it makes sense to extract common parts into some other `.sh` files? Like for example `create_wpios_pr.sh` and `create_wpandroid_pr.sh`.  I think this way we won't have to update both `./release_automation.sh` and `wp_gutenberg_ref_update_prs.sh` files when there's a change in one of the commands.